src/app/body/view-organization-details/members-access/members-access.component.html

```diff
@@ -36,7 +36,34 @@
                     <td class="align-middle">
                         <app-member-card [member] = "member"></app-member-card>
                     </td>
+                    <td class="align-middle" *ngIf="deleteAccess">
+                      <div class="btn" data-target="#deleteMember" data-toggle="modal" data-bs-toggle="tooltip">
+                        <mat-icon class="ml-mt-2" id="button" fontSet="material-icons-outlined">delete</mat-icon>
+                      </div>
+                    </td>
                   </tr>
+                  <div class="modal fade" id="deleteMember" tabindex="-1" role="dialog">
+                    <div class="modal-dialog modal-dialog-centered">
+                        <div class="modal-content">
+                            <div class="modal-header">
+                                <h2 class="modal-title" id="DeleteModalLabel">Are you sure you want to delete this user?</h2>
+                                <span class="btn">
+                                    <mat-icon fontSet="material-icons-outlined" class="align-middle" data-dismiss="modal" aria-label="Close">close</mat-icon>
+                                </span>
+                            </div>
+                            <div class="row p-m-2 text-center">
+                                <div class="col" style="font-size: medium;flex-wrap: wrap;"><h4> The user will be deleted from everywhere including Teams and Organisation if you delete.</h4></div>
+                            </div>
```

Comment:
Content can be improved
_Are you sure to delete the user? This Action will delete the user from all the teams the user is part of and cannot be undone._

Reply:
Did the required changes.